src/resolvers/git.cr

```diff
@@ -316,8 +333,9 @@ module Shards
       # be used interactively.
       # This configuration can be overriden by defining the environment
       # variable `GIT_ASKPASS`.
-      git_retry(err: "Failed to clone #{git_url}") do
-        run_in_current_folder "git clone -c core.askPass=true --mirror --quiet -- #{Process.quote(git_url)} #{Process.quote(local_path)}"
+      url = git_url.split("?").first
```

Comment:
This might need to be fixed if there _ARE_ query params that can be used in a clone. I think I've seen `shallow` used in the specs